politeiawww/www.go

```diff
@@ -912,6 +912,9 @@ func _main() error {
 	log.Infof("Network : %v", activeNetParams.Params.Name)
 	log.Infof("Home dir: %v", loadedCfg.HomeDir)
 
+	paywallAmountInDcr := float64(loadedCfg.PaywallAmount) / 1e8
+	log.Infof("Paywall amount in DCR is: %v dcr", paywallAmountInDcr)
```

Comment:
This is a little verbose, it can be shortened to:

    log.Infof("Paywall : %v DCR", paywallAmountInDcr)

That will also make it easier to read because the colons will line up with the rest of the output.